src/tests/test_utils/common_test_utils/include/common_test_utils/test_common.hpp

```diff
@@ -18,7 +18,7 @@ class TestsCommon;
 
 // Temporal WA for applications.ai.vpu-accelerators.vpux-plugin repo
```

Comment:
Please update `vpu` and `vpux` in comments as well

Reply:
This specific comment refers to the repo name - we will not be able to update the repo name in this round of renaming. Will search again for other comments.